clients/src/test/java/org/apache/kafka/common/UuidTest.java

```diff
@@ -72,12 +74,13 @@ public void testStringConversion() {
         assertEquals(Uuid.fromString(zeroIdString), Uuid.ZERO_UUID);
     }
 
-    @Test
+    @RepeatedTest(100)
```

Comment:
(63/64)^100 is ~20%.

Reply:
Yea, I figure if a bug was introduced to the Uuid generation, this test would eventually show up as a flaky failure and lead to investigation.